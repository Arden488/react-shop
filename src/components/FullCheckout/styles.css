.full-checkout {
  display: grid;
  grid-template-columns: auto 300px;
  grid-column-gap: 20px;
}

/* .full-checkout-summary {} */
