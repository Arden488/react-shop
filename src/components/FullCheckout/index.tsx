import * as React from 'react';

import './styles.css';

const FullCheckout = () => (
  <div>
    <h1>Checkout</h1>
    <p>Checkout</p>
  </div>
);

export default FullCheckout;
