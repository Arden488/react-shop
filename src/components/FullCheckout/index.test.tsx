import { shallow } from 'enzyme';
import * as React from 'react';

import FullCheckout from './index';

let component: React.ReactNode = null;

beforeEach(() => {
  component = shallow(<FullCheckout />);
});

it('renders correctly', () => {
  expect(component).toMatchSnapshot();
});
