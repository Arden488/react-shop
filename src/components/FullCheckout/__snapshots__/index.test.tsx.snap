// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <h1>
    Checkout
  </h1>
  <p>
    Checkout
  </p>
</div>
`;
